lib/commons/dom/index.js

```diff
@@ -40,7 +40,7 @@ export { default as isNode } from './is-node';
 export { default as isOffscreen } from './is-offscreen';
 export { default as isOpaque } from './is-opaque';
 export { default as isSkipLink } from './is-skip-link';
-export { default as isVisibleToScreenReaders } from './is-visible-for-screenreader';
+export { default as isVisibleToScreenReaders } from './is-visible-to-screenreader';
```

Comment:
I noticed that the file names were still using `for` instead of `to` so changed those to match the name of the function while I was there.